devops/cloudbuild-pr.yaml

```diff
@@ -0,0 +1,56 @@
+steps:
+  # install / setup ci
+  - name: node:16.14.2
+    entrypoint: npm
+    args: ['install']
+
+  # setup .env
+  - name: 'bash'
+    script: |
+      #!/usr/bin/env bash
+      cp .env.example .env
+
+  #
+  # Generate the static site
+  #
+  - name: node:16.14.2
+    entrypoint: npm
+    env:
+      - 'VUE_APP_ADDRESS_COMPLETE_KEY=$_VUE_APP_ADDRESS_COMPLETE_KEY'
+      - 'VUE_APP_BUSINESS_CREATE_LD_CLIENT_ID=$_VUE_APP_BUSINESS_CREATE_LD_CLIENT_ID'
```

Comment:
Why are only some keys included here?

Reply:
We set these keys (secret kinds or sensitive) in cloud build.

![image](https://user-images.githubusercontent.com/44277752/233497584-19bcada2-3884-4e8e-95dc-fc855528f46a.png)

Reply:
Other keys is in .env.example file.

Reply:
Aren't those values in 1Password?

I still don't understand why these 2 keys are in both the vaults.env file and in here -- if I add 1Password keys, how do I decide whether to add them here also?

Reply:
If the key is not secret value (such as url, path or non-sensitive) 
 You add the key and vaule in .env.example file for local development and Preview site. 

If the key is a secret value (such as key, token, password or don't want to expose to public) 
 You add the key name in .env.example file for local development.
 You add the key variable in cloudbuild-pr.yaml for Preview site.